x/bank/README.md

```diff
@@ -1032,3 +1036,13 @@ Example Output:
   }
 }
 ```
+
+## concepts/changes specific to heimdall-v2
+
+Since heimdall is a unique component of the PoS architecture and not a traditional Cosmos SDK app, some changes had to be made in order to ensure correct functionality, some of which are :
+
+* MsgSetSendEnabled is not supported since matic is the only denom used. 
+
+* (Un)delegation is not supported since staking related logic is handled by PoS [core contracts](https://github.com/0xPolygon/core-contracts/tree/main/contracts) deployed on Ethereum.
+
+* A default fee of 10^15 matic (`DefaultFeeInMatic`) is deducted from the tx sender.
```

Comment:
Shouldn't this be something that's handled in the antehandler and an expected default minimum fee in addition to gas?

Reply:
I also included this in the `auth` `README` just for redundancy. Hence, comment can be resolved IMHO.